src/Microsoft.AspNetCore.Sockets/Transports/ServerSentEventsTransport.cs

```diff
@@ -27,43 +29,43 @@ public ServerSentEventsTransport(ReadableChannel<Message> application, ILoggerFa
             context.Response.ContentType = "text/event-stream";
             context.Response.Headers["Cache-Control"] = "no-cache";
             context.Response.Headers["Content-Encoding"] = "identity";
-            await context.Response.Body.FlushAsync();
+
+            var pipe = context.Response.Body.AsPipelineWriter();
+
+            await pipe.Alloc().FlushAsync();
```

Comment:
Just like we used to do with the stream. I suppose I could do it with the Stream before adapting